lib/endpoints/class-wp-rest-controller.php

```diff
@@ -11,121 +11,121 @@ public function register_routes() {
 	}
 
 	/**
-	 * Get a collection of items
+	 * Get a collection of items.
 	 *
 	 * @param WP_REST_Request $request Full data about the request.
-	 * @return WP_Error|WP_REST_Response
+	 * @return WP_Error|WP_REST_Response.
```

Comment:
I haven't ever seen periods at the end of the type, only when there's a comment too. Can you clarify where else in core follows this pattern?